ocaml/src/tools/net_fd.ml

```diff
@@ -79,3 +79,13 @@ let read_all_lwt_bytes_exact target offset length = function
        (Lwt_ssl.read_bytes socket target)
        offset length
      >>= Lwt_extra2.expect_exact_length length
+
+
+let cork = function
+  | Plain fd -> Lwt_unix.setsockopt fd Lwt_unix.TCP_NODELAY false
+  | _ -> ()
+
+let uncork = function
+  | Plain fd -> Lwt_unix.setsockopt fd Lwt_unix.TCP_NODELAY true
+  | _ -> ()
```

Comment:
I'm of the opinion that one should only match on `_` in exceptional circumstances...
In this case here for the Ssl case we may want to call flush ( https://github.com/savonet/ocaml-ssl/blob/master/src/ssl.mli#L378-L379 ), however I did not find that call in Lwt_ssl?
It may be useful to do the same perf tests with asds using ssl.